### PR TITLE
Fix grammatical typos

### PR DESCRIPTION
**Pull request by mstfbl**
Fix grammatical typos. Utilized [Visual Studio Spell Checker](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater) to quickly find and implement typo fixes.

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public static void Example()
                );
 
             // Load the TensorFlow model once.
-            //      - Use it for quering the schema for input and output in the
+            //      - Use it for queering the schema for input and output in the
```
Comment (gvashishtha) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: querying?

**Comment by mstfbl** [comment]
@gvashishtha Visual Studio Spell Checker actually marked certain correctly spelled words that are not in its dictionary as misspelled, so the human factor here is necessary to ascertain when the Spell Checker is correct or not. As such I don't think it would be realistically viable to have this as part of the CI build check.

**Comment by codemzs** [comment]
@gvashishtha The point of this PR is to fix typos(see the title) not enable tests as part of CI build so blocking PR on it for that reason is not acceptable. @mstfbl feel free to dismiss the block.

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs (hunk 1 above)
Comment (lizelive) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: queering the schema 🏳️‍🌈

**Pull request merged**